eng/Directory.Build.Common.targets

```diff
@@ -188,7 +188,15 @@
   
   <!--TODO: end-->
 
-  <!-- *********** Management Client Library Override section ************* -->
+  <!-- *********** Files needed for LRO ************* -->
+  <ItemGroup Condition="'$(IncludeOperationsSharedSource)' == 'true'">
+    <Compile Include="$(AzureCoreSharedSources)OperationHelpers.cs" LinkBase="Shared/Core" />
```

Comment:
Any particular reason we are moving the link from `Shared` to `Shared/Core`. Are we planning to have a hierarchy with different files at different levels.

Reply:
The list of files for lro includes will be changing a few times as we do some refactoring so this makes it easier to make those changes as we move along

Reply:
I think @Mohit-Chakraborty 's question was why we didn't use `LinkBase="Shared"` like the other shared source files.

Reply:
Correct. If we are going to have other subfolders as links, it makes sense.

Reply:
Should all projects using only `LinkBase="Shared"` be updated to use `LinkBase="Shared/Core"` then so all files are visually co-located?